include/seqan/sequence/string_packed.h

```diff
@@ -585,6 +589,76 @@ resize(
     return front(host(me)).i = new_length;
 }
 
+template <typename TValue, typename THostspec, typename TSize, typename TExpand, typename TValue2>
+inline typename Size< String<TValue, Packed<THostspec> > >::Type
+resize(String<TValue, Packed<THostspec> > & me,
+       TSize new_length,
+       TValue2 const & newValue,
+       Tag<TExpand> tag)
+{
+    typedef String<TValue, Packed<THostspec> > TString;
+    typedef PackedTraits_<TString> TTraits;
+    typedef typename Size<TString>::Type TStringSize;
+    typedef typename TTraits::THostValue THostValue;
+
+    TStringSize old_length = length(me);
+    TStringSize old_host_length = length(host(me));
+
+    if (static_cast<TStringSize>(new_length) < length(me))
+        return resize(me, new_length, tag);
+
+    // create WORD
+    THostValue tmp;
+    tmp.i = 0ul;
+    __uint64 ord = ordValue(TValue(newValue));
+    for (unsigned j = 0; j < TTraits::VALUES_PER_HOST_VALUE; ++j)
+        tmp.i |= ord << (j * TTraits::BITS_PER_VALUE);
+
+    // fill up host string word-wise
+    TStringSize max_length = (resize(host(me),
+                                     TTraits::toHostLength(new_length) + 1,
+                                     tmp,
+                                     tag) - 1)
+                             * TTraits::VALUES_PER_HOST_VALUE;
+
+    // set values on formerly last word, because not affected by above
+    if (old_host_length > 1)
+    {
+        TStringSize alreadySet = old_length % TTraits::VALUES_PER_HOST_VALUE;
+
+        // clear non-set positions (which may be uninitialized ( != 0 )
+        tmp.i = (~0ul << (64 - alreadySet * TTraits::BITS_PER_VALUE)) >> TTraits::WASTED_BITS;
+        host(me)[old_host_length-1].i &= tmp.i;
+
+        for (TStringSize i = alreadySet; i < TTraits::VALUES_PER_HOST_VALUE; ++i)
+            host(me)[old_host_length-1].i |=  ord << ((TTraits::VALUES_PER_HOST_VALUE - i - 1) * TTraits::BITS_PER_VALUE);
```

Comment:
> I think I remember it to be from MSB to LSB.

It is `[ WASTED_BITS | FIRST_VALUE | SECOND_VALUE | ... ]` and inside the values it is MSB to LSB that is correct. Bus since I am shifting in an entire pattern in this is the way to go.